cmd/osv-scanner/callanalysis_parser.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+var stableCallAnalysisStates = map[string]bool{
+	"go":   true,
+	"rust": false,
+}
+
+// Creates a map to record if languages are enabled or disabled for call analysis.
+func createCallAnalysisStates(enabledCallAnalysis []string, disabledCallAnalysis []string) map[string]bool {
+	enableAll, disableAll := false, false
```

Comment:
you could avoid the conditions in the first two loops by expecting "all" in the actual `callAnalysisStates` - while I think you'd still need to have a variable outside the third loop, I feel like it would still be simpler (effectively it should let you shift all the logic to the end of the function) 🤔 

(not a blocker, just worth playing with if you havn't already)

Reply:
Thanks Gareth! Updated.